src/inputs/radio/radio.jsx

```diff
@@ -121,7 +177,7 @@ const styles = (theme) => {
                 color: theme.palette.text.primary,
                 cursor: 'pointer',
                 display: 'block',
-                fontSize: 14,
+                fontSize: 16,
```

Comment:
Please change to `typography.pxToRem(16)`

Reply:
done!